core/src/common.cpp

```diff
@@ -401,9 +401,20 @@ namespace irods::http
 		// Verify the bearer token is known to the server. If not, return an error.
 		auto mapped_value{irods::http::process_stash::find(bearer_token)};
 		if (!mapped_value.has_value()) {
+			const auto& config = irods::http::globals::configuration();
+
+			// It's possible that the admin didn't include the OIDC configuration stanza.
+			// This use-case is allowed, therefore we check for the OIDC configuration before
+			// attempting to access it. Without this logic, the server would crash.
+			const auto oidc_config_iter =
+				config.find(nlohmann::json::json_pointer{"/http_server/authentication/openid_connect"});
+			if (oidc_config_iter == std::end(config)) {
+				logging::trace("{}: No openid_connect configuration found in server configuration.", __func__);
```

Comment:
I'm thinking `debug` may be slightly better here, but I'm also fine with `trace`.

Reply:
I don't mind using debug. Will update.